testing/bdd/templates/timeBase_clock_clockMode_mandatory.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+<tt:tt
+        ebuttp:sequenceIdentifier="testSequence001"
+        ebuttp:sequenceNumber="1"
+        ttp:timeBase="{{ time_base }}"
+{% if clock_mode %}
+  {% if clock_mode == "*?Empty?*" %}
+        ttp:clockMode=""
```

Comment:
Previous comment about empty `clockMode` related to this line. On second thoughts, should this line deliberately omit adding the `clockMode` attribute, rather than adding an empty one?